.github/workflows/release.yml

```diff
@@ -31,10 +31,12 @@ jobs:
           IFS=""
           echo "$PGP_PRIVATE_KEY" | gpg --dearmor > $HOME/secring.gpg
           echo "$PGP_PASSPHRASE" > $HOME/passphrase.txt
+          echo "$PGP_PUBLIC_KEY" > $HOME/pubring.gpg
```

Comment:
Where is `PGP_PUBLIC_KEY` set?

Reply:
It is added via the actions secrets

Reply:
We could also download it from keybase.io

Reply:
In that case, I think it must be added to `env` to be available.